src/Storefront/Resources/views/storefront/page/product-detail/headline.html.twig

```diff
@@ -17,6 +17,8 @@
                     {% block page_product_detail_manufacturer_link %}
                         <a href="{{ page.product.manufacturer.link }}"
                            class="product-detail-manufacturer-link"
+                           rel="noreferrer"
+                           rel="noopener"
```

Comment:
You can't have two rel. It needs to be in one with whitespace seperated

Reply:
I'll fix that after the recap

Reply:
Fixed